backend/app/serializers/api/v1/funder_serializer.rb

```diff
@@ -20,11 +14,9 @@ class FunderSerializer < BaseSerializer
         :capital_acceptances_other,
         :leadership_demographics,
         :leadership_demographics_other,
-        :number_staff_employees,
         :application_status,
         :funder_legal_status,
         :funder_legal_status_other,
-        :new_to_regenerative_ag,
```

Comment:
I have removed some attributes from Funder API which were `ONLY FORA` @mbarrenechea . To name them all:
```
:primary_office_address,
:primary_contact_first_name,
:primary_contact_last_name,
:primary_contact_phone,
:primary_contact_location,
 :primary_contact_role,
:number_staff_employees,
:new_to_regenerative_ag
```
Please make sure that some part of FE is not using them.

Reply:
`primary_office_address` is used I think, couldn't find the others. it is used I think in an interface definition, so maybe it won't blow up?